scrappers/codeforces/scrapper.go

```diff
@@ -28,20 +29,24 @@ func (s Scrapper) GetProfileInfo() types.ProfileInfo {
 	}
 	var profile types.ProfileInfo
 	url := "http://codeforces.com/api/user.info?handles=" + s.Handle
-	data, _ := common.HitGetRequest(url)
-	if data == nil {
-		log.Println(errors.New("GetRequest failed. Please check connection status"))
-		hub.CaptureException(errors.New("GetRequest failed. Please check connection status"))
-		return types.ProfileInfo{}
-	}
-	err := json.Unmarshal(data, &profile)
-	if err != nil {
-		log.Println(err.Error())
+	for attempt := 1; attempt < 10; attempt++ {
+		time.Sleep(time.Second * time.Duration(attempt))
+		log.Println(attempt)
+		data, _ := common.HitGetRequest(url)
+		if data == nil {
+			log.Println(errors.New("GetRequest failed. Please check connection status"))
+			hub.CaptureException(errors.New("GetRequest failed. Please check connection status"))
+			return types.ProfileInfo{}
+		}
+		err := json.Unmarshal(data, &profile)
+		if err == nil {
+			break
+		}
 		hub.AddBreadcrumb(&sentry.Breadcrumb{
 			Category: "JSON parse error",
 			Message:  string(data),
 		}, nil)
-
+		log.Println(err.Error())
 		hub.CaptureException(err)
```

Comment:
remove this from here, and add this just after for loop :
`if err != nil {
		hub.CaptureException(err)
	}`

Reply:
you may need to create variable `err` before the loop